ecl/hthor/hthor.cpp

```diff
@@ -8064,16 +8064,21 @@ void CHThorDiskReadBaseActivity::ready()
     if (ldFile)
         dFile = ldFile->queryDistributedFile();  // Null for local file usage
 
-    Linked<IOutputMetaData> publishedMeta = actualDiskMeta;
-    unsigned publishedCrc = actualCrc;
+    Owned<IOutputMetaData> publishedMeta;
```

Comment:
What is publishedMeta initialized to ?  Previously it was to actualDiskMeta.

Reply:
It is not.
It is intentionally left empty, unless format matches